### ISSUE TITLE
Need of other Pre-trained weight of mobilenet(width_multiplier=[0.25, 0.5, 0.75])

### ISSUE THREAD

**Issue by nieyan** [Zehaos/MobileNet#14]
Hello,
thanks a lot for releasing the implementation of MobileNets!
I have downloaded the mobilenet_width_1.0 and used the head of the model weights as the init for other task. And it perform well !  I am training the mobilenet(width_multiplier=[0.25, 0.5, 0.75]) in imagenet data to get the Pre-trained weight because the model size and inference time.  But loss converge too slow. Would you please release the weight of the other width_multiplier if you have trained before? :)

Thanks!

**Comment by Zehaos** [comment]
Hi, nieyan

I haven't trained mobilenet width_multiplier in [0.25, 0.5, 0.75] due to limited resource.

**Comment by nieyan** [comment]
Thanks for your quick reply! I will try to use multi-gpu during training. :)

**Comment by ydzhang12345** [comment]
@nieyan Hi, I am looking for the same things too! Have you managed to train mobilenet with different width multiplier?

**Comment by nieyan** [comment]
Google Released Pre-Trained MobileNets
https://www.infoq.com/news/2017/06/google-mobilenets-tensorflow
